ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -118,73 +119,102 @@ class Endpoints(
     }
   }
 
+  def withJwtPayloadLoggingContext[A, B](jwtPayload: JwtPayload)(
+      fn: LoggingContextOf[JwtPayload with InstanceUUID with RequestID] => A
+  )(implicit lc: LoggingContextOf[InstanceUUID with RequestID]): A =
```

Comment:
If you want to determine `B` inferrably, a label can sit alongside `jwtPayload`.

```suggestion
      fn: LoggingContextOf[JwtPayload with B] => A
  )(implicit lc: LoggingContextOf[B]): A =
```

Reply:
True, this would be better. Thanks!